kong/core/balancer.lua

```diff
@@ -603,10 +605,29 @@ end
 
 
 local function init()
+
+  if balancers_initialized then
+    return true
+  end
+
+  local lock, err = resty_lock:new("kong", { timeout = 3600 })
```

Comment:
Isn't this timeout a bit excessive? Something like `30` might be safer? Is there a particular intent why it is so high? Anyway, it doesn't make much sense for it to be higher than the default `exptime` (which is 30s), and actually, it would be modified to `30` right now, see https://github.com/openresty/lua-resty-lock/blob/master/lib/resty/lock.lua#L104